.github/workflows/lint.yml

```diff
@@ -0,0 +1,59 @@
+name: lint
+
+on:
+  push:
+    branches:
+    - develop
+    - stable
+    - 'GSOC**'
+  pull_request:
+    branches:
+    - develop
+    - stable
+    - 'GSOC**'
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.10"
+    - name: Lint with flake8
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8
```

Comment:
flake8-builtins is missing

Reply:
oh, good eyes :)

Reply:
Just FYI, in this particular case `git diff --ignore-all-space --ignore-blank-lines --color-moved develop` is very helpful. This line with the missing package stands out pretty well in the middle of a code block that is otherwise moved only.